yarn-project/foundation/src/serialize/serialize.ts

```diff
@@ -168,6 +185,29 @@ export function serializeToBufferArray(...objs: Bufferable[]): Buffer[] {
   return ret;
 }
 
+/**
+ * Serializes a list of objects contiguously.
+ * @param objs - Objects to serialize.
+ * @returns An array of fields with the concatenation of all fields.
+ */
+export function serializeToFieldArray(...objs: Fieldeable[]): Fr[] {
```

Comment:
As mentioned earlier I think we could likely be using this together with the `GetFields` that some of the types already had.